app/scripts/modules/netflix/tableau/summary/summaryTableau.controller.ts

```diff
@@ -0,0 +1,26 @@
+import {ISCEService, module} from 'angular';
+
+import {AUTHENTICATION_SERVICE, AuthenticationService} from 'core/authentication/authentication.service';
+import {NetflixSettings} from '../../netflix.settings';
+
+import '../tableau.less';
+
+class SummaryTableauController {
+
+  public srcUrl: string;
+
+  static get $inject(): string[] {
+    return ['$sce', 'app', 'authenticationService'];
```

Comment:
I clicked the analytics tab, waited for all the UI graphs to load, and confirmed that the URL requested in the network stack matched the one requested by the controller.  Not sure how it got changed between testing and committing.  Apologies.  Thanks @christopherthielen for catching and @jrsquared for fixing.